src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution_ArgsToParameters.cs

```diff
@@ -55,6 +55,34 @@ public ImmutableArray<int> ToImmutableArray()
             }
         }
 
+        internal static ImmutableArray<ParameterSymbol> GetParametersIncludingExtensionParameter(Symbol symbol)
+        {
+            Debug.Assert(symbol.GetIsNewExtensionMember());
+            // PROTOTYPE consider optimizing
+            return [symbol.ContainingType.ExtensionParameter, .. symbol.GetParameters()];
+        }
+
+        private static ImmutableArray<RefKind> GetParameterRefKindsIncludingReceiver(Symbol symbol)
+        {
+            Debug.Assert(symbol.GetIsNewExtensionMember());
+            // PROTOTYPE consider optimizing
+            var refKinds = symbol.GetParameterRefKinds();
+            var receiverRefKind = symbol.ContainingType.ExtensionParameter.RefKind;
+            if (refKinds.IsDefault)
+            {
+                return receiverRefKind == RefKind.None ? default : [receiverRefKind];
```

Comment:
>[receiverRefKind] [](http://example.com/codeflow?start=67&length=17)

This looks suspicious. It looks like ```symbol.GetParameterRefKinds()``` can returns default when all parameters are non-ref. I think this array is supposed to include information for all "real" parameters too (RefKind.None).